orga/tests/integration/components/campaign/list_test.js

```diff
@@ -283,5 +283,61 @@ module('Integration | Component | Campaign::List', function (hooks) {
         assert.dom(screen.queryByLabelText('Dupont Alice')).doesNotExist();
       });
     });
+
+    module('clear filters button', function () {
+      test('it should be disabled', async function (assert) {
```

Comment:
**suggestion** : peut-être rajouter un peu de contexte pour dire quand est-ce que ça devrait être enabled et disabled, genre "it should be disabled when ..."